### PR TITLE
glava: 1.4.5 -> 1.5.1

### PR DESCRIPTION
**Pull request by eadwu**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/glava/default.nix [hunk 1]
```diff
@@ -43,12 +33,12 @@ let
 in
   stdenv.mkDerivation rec {
     name = "glava-${version}";
+    version = "1.5.1";
 
-    src = fetchFromGitHub {
-      owner = "wacossusca34";
-      repo = "glava";
+    src = fetchgit {
```
Comment (srhb) [inline pkgs/applications/misc/glava/default.nix]: Why did you change this to fetchgit?

Comment (eadwu) [reply pkgs/applications/misc/glava/default.nix]: I didn't seem to get the glad submodule when I used fetchFromGithub, since the current revision in the repository works now.

**Comment by srhb** [comment]
@GrahamcOfBorg build glava
